types/audit-details.d.ts

```diff
@@ -51,7 +51,7 @@ declare global {
         overallSavingsBytes?: number;
         headings: OpportunityColumnHeading[];
         items: OpportunityItem[];
-        diagnostic?: Diagnostic;
+        diagnostic?: DebugData;
```

Comment:
I think all these `diagnostic` need to be renamed?

Reply:
yeah fo sho, didn't expect the first name go around to be accepted :D